services/movie_session.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    movies_sessions = MovieSession.objects.all()
+    if session_date:
+        movies_sessions = movies_sessions.filter(
+            show_time__date=session_date
+        )
+    return movies_sessions
+
+
+def get_movie_session_by_id(
+        movie_session_id: int
+) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: str = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> MovieSession:
+    movie_session = MovieSession.objects.get(id=session_id)
+    if show_time:
+        movie_session.show_time = show_time
+    if movie_id:
+        movie_session.movie_id = movie_id
+    if cinema_hall_id:
+        movie_session.cinema_hall_id = cinema_hall_id
+    return movie_session.save()
+
+
+def delete_movie_session_by_id(
+        session_id: int
+) -> MovieSession:
+    deleted_session = MovieSession.objects.get(id=session_id)
+    deleted_session.delete()
+    return deleted_session
```

Comment:
You could make it in one line without returns . 
That is for future : )
`MovieSession.objects.get(id=session_id).delete()`